### PR TITLE
move underscore to its old place and remove unnecessary packages

### PR DESCRIPTION
**Pull request by webzwo0i**
for https://github.com/ether/etherpad-lite/issues/4875

I think this is better. The underscore wrapper again lives at ./src/static/js and not in vendors directory, as it's really just a wrapper. If we don't do this, all plugins that use underscore from core would need to be updated anyway. So if we really want to move that file we should probably include a warning in the plugin devs helper script for a while?

It also removes the imo unnecessary packages in tar.json. Checking the filename in Minify.js only helps plugins when they link to `/static/js/jquery.js` in their script tags. If they use jquery with `require` they should always use rjquery - which is still at its old place.

@rhansen what do you think? Did I miss something fundamental?

### CODE REVIEW

File: src/node/utils/Minify.js [hunk 1]
```diff
@@ -141,19 +141,11 @@ const minify = async (req, res) => {
 
   // Backward compatibility for plugins that were written when jQuery lived at
   // src/static/js/jquery.js.
-  if (['js/jquery.js', 'plugins/ep_etherpad-lite/static/js/jquery.js'].indexOf(filename) !== -1) {
+  if (filename.endsWith('js/jquery.js')) {
```
Comment (rhansen) [inline src/node/utils/Minify.js:144]: This matches `foojs/jquery.js`, which I don't think is a good idea. I think it's better to leave this line as-is.

File: src/node/utils/tar.json [hunk 2]
```diff
@@ -82,6 +82,4 @@
   , "pluginfw/shared.js"
   , "pluginfw/hooks.js"
   ]
-, "jquery.js": ["jquery.js"]
```
Comment (rhansen) [inline src/node/utils/tar.json:85]: I coulda swore this line was necessary to support plugins that have frontend code that does `require('jquery')`, but maybe not.

**Review by rhansen — APPROVED** [review]

File: src/node/utils/Minify.js (hunk 1 above)
Comment (webzwo0i) [reply src/node/utils/Minify.js:144]: OK good point

File: src/node/utils/tar.json (hunk 2 above)
Comment (webzwo0i) [reply src/node/utils/tar.json:85]: I would expect `require('ep_etherpad-lite/static/js/jquery')` in this case, which will work and `require('jquery')` to resolve to a local dependency of the plugin.

**Pull request merged**